MTCircularSlider/Classes/MTCircularSlider.swift

```diff
@@ -46,6 +46,8 @@ public enum Attributes {
 	case thumbRadius(CGFloat)
 	case thumbShadowRadius(CGFloat)
 	case thumbShadowDepth(CGFloat)
+    case thumbBorderWidth(CGFloat)
```

Comment:
in my xcode the indentation looks correct. maybe my indentation preferences have to match yours? Not sure